src/applications/appeals/testing/components/ReviewPage2.jsx

```diff
@@ -0,0 +1,152 @@
+        props.goToPath('/confirmation');
+      }
+    },
+  };
+
+  return (
+    <article>
+      <div name="topScrollElement" />
+      <div name="topNavScrollElement" />
+      <h1>Review Board Appeal</h1>
+      <va-on-this-page uswds />
+      {chapterTitles.filter(title => title !== 'Apply').map((title, index) => (
+        <div key={index}>
+          <div className={chapterClasses}>
+            <h2 id={index} className="vads-u-margin--0">
+              {title}
+            </h2>
+          </div>
+          <ul className="review-pages vads-u-padding--0">
+            {getChapterPagesFromChapterIndex(index).map(page => {
+              const depends = page.depends ? page.depends(props.data) : true;
+              return page.review && depends && !page.taskListHide
+                ? Object.entries(page.review(props.data)).map(
+                    ([label, value], rowIndex) => (
+                      <li key={label} className={listClasses}>
+                        <div className={divClasses}>
+                          <div>
+                            <div className="page-title vads-u-color--gray">
+                              {label}
+                            </div>
+                            <div className="page-value">{value}</div>
+                          </div>
+                          {/* Not ideal, but for now, only render one edit link
+                           for the contact info page */}
+                          {rowIndex > 0 ? null : (
+                            <Link to={page.path}>
+                              Edit{' '}
+                              <span className="vads-u-visibility--screen-reader">
+                                {page.title}
+                              </span>
+                            </Link>
+                          )}
+                        </div>
+                      </li>
+                    ),
+                  )
+                : null;
+            })}
+          </ul>
+        </div>
+      ))}
+      <p className="vads-u-margin-top--6">
+        <strong>Note:</strong> According to federal law, there are criminal
+        penalties, including a fine and/or imprisonment for up to 5 years, for
+        withholding information or for providing incorrect information. (See 18
+        U.S.C. 1001)
+      </p>
+      <VaPrivacyAgreement
+        onVaChange={handlers.onPrivacyCheckboxChange}
+        showError={submitted && !privacyCheckbox}
+        uswds
+      />
+      <p className="vads-u-margin-top--4">
+        <Link to="/decision-reviews/appeals-testing">
+          Finish this application later
+        </Link>
+      </p>
+      {/* {props.contentBeforeButtons} */}
+      <va-button onClick={handlers.onSubmit} text="Submit" />
+      <FormNavButtons goBack={props.goBack} goForward={handlers.onSubmit} />
+      {/* {props.contentAfterButtons} */}
+    </article>
+  );
+};
+
+ReviewPage.propTypes = {
+  contentAfterButtons: PropTypes.element,
```

Comment:
are these needed along with `goBack` and `goForward` `propTypes`?

Reply:
Only for apps that require the Veteran/user to be logged in

Reply:
And I don't think it'd hurt to leave the `PropTypes` there 🤷🏽